nucypher/characters/chaotic.py

```diff
@@ -274,16 +301,20 @@ def home():
         @limiter.limit("5 per day")
```

Comment:
@vepkenez Do we still want this limitation? Does this limit to 5 recipients per day?